src/collective/cover/tiles/list.py

```diff
@@ -221,7 +219,6 @@ def populate_with_uuids(self, uuids):
 
         old_data['uuids'] = uuids_dict
         data_mgr.set(old_data)
-        notify(ObjectModifiedEvent(self))
```

Comment:
No need to notify here, since ``data_mgr.set`` method already notify the changes.